dm/dm/master/server_test.go

```diff
@@ -244,7 +244,7 @@ func mockRevelantWorkerClient(mockWorkerClient *pbmock.MockWorkerClient, taskNam
 			expect = pb.Stage_Running
 		case pb.TaskOp_Pause:
 			expect = pb.Stage_Paused
-		case pb.TaskOp_Stop:
+		case pb.TaskOp_Delete:
```

Comment:
here too

Reply:
op_delete will delete this subtask on worker, so there is not a finished stage
only two sences have  finished stage: dump finished and load finished